nationalarchives/ds-wagtail
# [UN-532] Large link card metrics

Ticket URL: https://national-archives.atlassian.net/browse/UN-532

## About these changes

Updates 'Card title' to 'Card text' for `data-link-type` 

## How to check these changes

1. Check record revealed locally and inspect the large link cards
2. See that the card is showing 'Card text' instead of 'Card title'

## Before assigning to reviewer, please make sure you have


## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)
